### DISCUSSION TITLE
Sponsorship from Avalonia Chinese community members

### DISCUSSION THREAD

**Discussion by rabbitism** [AvaloniaUI/Avalonia#11467] [General]
Dear All, 
  Many Avalonia users and developers in Chinese community want to sponsor AvaloniaUI but lack of convenient payment. On behalf of their kindness, I help and gather to sponsor via @avaloniachina account but all credits to these amazing community members!
Here is the list of sponsorship.
  许多Avaolnia中文社区的开发者想要通过赞助的形式支持AvaloniaUI项目，但苦于没用合适的支付渠道。因此我帮助他们通过avaloniachina账号进行了赞助。感谢每一位赞助者！

![img_v2_1e5c43c2-1d8e-4815-aaba-2c71cd23cddg](https://github.com/AvaloniaUI/Avalonia/assets/14807942/338d7bf1-08a0-4eac-b580-a07b09ffe3f2)

|Name/alias|GithubId|amount|
-- | -- | --
霍小平 | @mrhuo | ¥100.00
Rwing | @Rwing | ¥11.00
Neo | @my522cn | ¥200.00
liuyon9 |   | ¥8.00
老吴 | @wj8400684 | ¥500.00
闫志刚 | @kesshei | ¥100.00
吴腾飞 | @xiaobaiafei | ¥100.00
吴锋 | @ChisterWu | ¥100.00
匿名 |   | ¥20.00
万永红 |   | ¥50.00
韩伟朝 | @996666925 | ¥0.10
  | @afunc233 | ¥50.00
  | @yangjieshao | ¥100.00
Jason | @jason-zhang-cn | ¥100.00
李大勇 | @diyer03th | ¥100.00

**Comment by timunie** [comment]
While I am really happy to see this happening ♥️, I also wonder which option for sponsoring would help you in china. Is there any global available, that one may want to check out?

🙏Many thanks to all sponors 🙏

**Comment by kesshei** [comment]
The commonly used payment methods in China are Alipay and WeChat payment.
